R/utils.r

```diff
@@ -124,7 +124,19 @@ first_upper <- function(x) {
 
 download <- function(path, url, ...) {
   request <- httr::GET(url, ...)
-  httr::stop_for_status(request)
+
+  if (httr::status_code(request) >= 400) {
+    # Parse the response if it is from GitHub, use stop_for_status otherwise.
+    tryCatch(github_response(request),
```

Comment:
I guess the issue was what happens when the response is not from GitHub but somewhere else and is not JSON but another format? Then 
```
text <- httr::content(req, as = "text")
parsed <- jsonlite::fromJSON(text, simplifyVector = FALSE)
```
Is going to throw an error, but in that case we don't care about the parsing error, we just want to return the stop_for_status of the response.